modules/data.mining/inst/SpectralSlice.R

```diff
@@ -57,7 +57,7 @@ NEEm.sd <- NA
 if (is.nan(NEEm.bar)) {
   NEEm.bar <- NA
 } else {
-  NEEm.sd <- sqrt(var(NEEm, na.rm = TRUE))
+  NEEm.sd <- sqrt(nc_var(NEEm, na.rm = TRUE))
```

Comment:
revert

Reply:
@mdietze if you could explain why are we reverting these. isn't `var` used here an internal function? just a bit confused here.

Reply:
This `var` is `stats::var`. Someone was sloppy on namespaces (and given the filename, that was probably me over a decade ago when we were less strict about namespaces) and because `stats` is a core namespace to R the language is able to find the function.